code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KuptsovaAlesyaTest.java

```diff
@@ -0,0 +1,48 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+
+import static java.lang.String.valueOf;
+import static junit.framework.TestCase.assertEquals;
+
+public class KuptsovaAlesyaTest {
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void canNotCreateComplexNumberWithUncorrectValues() {
+        String re = "3.14";
+        String im = "-rjfho;i";
+        thrown.expect(IllegalArgumentException.class);
```

Comment:
can you rewrite test to somethink like that:
    @Test(expected = IllegalArgumentException.class)
    public void canNotCreateComplexNumberWithUncorrectValues() {
          your_test
    }